README.rst

```diff
@@ -36,6 +37,39 @@ Example of AIOKafkaProducer usage:
         loop.close()
 
 
+AIOKafkaConsumer
+****************
+
+AIOKafkaConsumer is a high-level, asynchronous message consumer.
+It also interacts with the assigned kafka Group Coordinator nodeto allow multiple consumers to load balance consumption of topics (requires kafka >= 0.9.0.0).
```

Comment:
`It interacts with the assigned kafka Group Coordinator node to allow multiple consumers to load balance consumption of topics (requires kafka >= 0.9.0.0).`